Content.Server/BluespaceHarvester/BluespaceHarvesterComponent.cs

```diff
@@ -0,0 +1,129 @@
+using Content.Shared.BluespaceHarvester;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.BluespaceHarvester;
+
+[RegisterComponent]
+public sealed partial class BluespaceHarvesterComponent : Component
+{
+    public bool Reseted = false;
+
+    /// <summary>
+    /// The current level at which the harvester is located is what other parameters are calculated from.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

Comment:
`ViewVariables(VVAccess.ReadWrite)`
is a standard parameter, and does not need to be declared. You can just write `[DataField]` if you not change Access type